# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ldapdns/ldapdns-2.06-gcc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur ldapdns-2.06.orig/ldapdns.c ldapdns-2.06/ldapdns.c
--- ldapdns-2.06.orig/ldapdns.c	2004-12-31 03:48:24.000000000 +0100
+++ ldapdns-2.06/ldapdns.c	2004-12-31 16:00:29.000000000 +0100
@@ -20,7 +20,7 @@
 		}
 		list_push(&x, lp->str);
 		list_push(&seen, lp->str);
-HIT:
+HIT:lp=lp;
 	}
 
 	lp = *p;
